### ISSUE TITLE
 undefined method `each' for nil:NilClass

### ISSUE THREAD

**Issue by GlebBelyaev** [fastlane/fastlane#13959]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

 Somebody with building project 🤷‍♂️ 

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>🔗  You can ⌘ + double-click on links to open them directly in your browser.
/Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/gym/lib/gym/error_handler.rb:147:in `print_full_log_path': [!] undefined method `each' for nil:NilClass (NoMethodError)
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/gym/lib/gym/error_handler.rb:59:in `handle_build_error'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/gym/lib/gym/runner.rb:97:in `block in build_app'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane_core/lib/fastlane_core/command_executor.rb:89:in `execute'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/gym/lib/gym/runner.rb:93:in `build_app'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/gym/lib/gym/runner.rb:18:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/gym/lib/gym/manager.rb:19:in `work'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/actions/build_ios_app.rb:64:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/runner.rb:253:in `block (2 levels) in execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/runner.rb:232:in `block in execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/runner.rb:227:in `chdir'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/runner.rb:227:in `execute_action'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/fast_file.rb:154:in `method_missing'
	from Fastfile:29:in `block (2 levels) in parsing_binding'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/lane_manager.rb:56:in `cruise_lane'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
	from /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/commands_generator.rb:333:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/fastlane/lib/fastlane/cli_tools_distributor.rb:108:in `take_off'
	from /Library/Ruby/Gems/2.3.0/gems/fastlane-2.112.0/bin/fastlane:23:in `<top (required)>'
	from /usr/local/bin/fastlane:22:in `load'
	from /usr/local/bin/fastlane:22:in `<main></pre>
</details>

### Environment

<details>
  <pre>### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.112.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.0.0        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.3        |
| CFPropertyList            | 3.0.0        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.7.0        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.4.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.4.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.2        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.6.5        |
| tty-cursor                | 0.6.0        |
| tty-spinner               | 0.9.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.62.0       |
| faraday                   | 0.15.4       |
| unf_ext                   | 0.0.7.5      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20180417 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.5        |
| gh_inspector              | 1.1.3        |
| mini_magick               | 4.5.1        |
| multi_json                | 1.13.1       |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.2        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.5.0        |
| bundler                   | 1.17.2       |
| faraday_middleware        | 0.12.2       |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.5        |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| mime-types-data           | 3.2018.0812  |
| mime-types                | 3.2.2        |
| jwt                       | 2.1.0        |
| signet                    | 0.11.0       |
| memoist                   | 0.16.0       |
| os                        | 1.0.0        |
| googleauth                | 0.6.7        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.23.9       |
| google-cloud-env          | 1.0.5        |
| google-cloud-core         | 1.2.7        |
| digest-crc                | 0.4.1        |
| google-cloud-storage      | 1.15.0       |
| emoji_regex               | 0.1.1        |
| json                      | 2.1.0        |
| io-console                | 0.4.5        |
| libxml-ruby               | 2.9.0        |
| psych                     | 2.1.0.1      |
</details>
</pre>
</details>

**Comment by minuscorp** [comment]
Can you provide some more information about the issue, the stack trace does not provide lots of clues of what's happening to you

**Comment by alexlanz** [comment]
I had the same issue today. The problem for me was that I was using the ruby version that I installed with Homebrew (/usr/local/bin/ruby). I uninstalled this version and used the one from MacOS (/usr/bin/ruby).

You can simply uninstall this version by running:

```bash
brew uninstall ruby
```

And if you are using the bundler with fastlane, you have to reinstall it after uninstalling ruby, because otherwise it points to the old ruby version:

```bash
gem install bundler
```

**Comment by minuscorp** [comment]
`RVM` or `rbenv` are always welcome, better than Apple's stock Ruby's version

**Comment by alexlanz** [comment]
But the Apple's stock Ruby version is working for me! That was strange for me..

**Comment by GlebBelyaev** [comment]
Thanks for your advice. I attached my gem env
`RubyGems Environment:
  - RUBYGEMS VERSION: 2.5.2.3
  - RUBY VERSION: 2.3.7 (2018-03-28 patchlevel 456) [universal.x86_64-darwin18]
  - INSTALLATION DIRECTORY: /Library/Ruby/Gems/2.3.0
  - USER INSTALLATION DIRECTORY: /Users/glebdev/.gem/ruby/2.3.0
  - RUBY EXECUTABLE: /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby
  - EXECUTABLE DIRECTORY: /usr/local/bin
  - SPEC CACHE DIRECTORY: /Users/glebdev/.gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /Library/Ruby/Site
  - RUBYGEMS PLATFORMS:
    - ruby
    - universal-darwin-18
  - GEM PATHS:
     - /Library/Ruby/Gems/2.3.0
     - /Users/glebdev/.gem/ruby/2.3.0
     - /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/gems/2.3.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /usr/local/bin
     - /usr/bin
     - /bin
     - /usr/sbin
     - /sbin
`

**Comment by GlebBelyaev** [comment]
> I had the same issue today. The problem for me was that I was using the ruby version that I installed with Homebrew (/usr/local/bin/ruby). I uninstalled this version and used the one from MacOS (/usr/bin/ruby).
> 
> You can simply uninstall this version by running:
> 
> ```shell
> brew uninstall ruby
> ```
> 
> And if you are using the bundler with fastlane, you have to reinstall it after uninstalling ruby, because otherwise it points to the old ruby version:
> 
> ```shell
> gem install bundler
> ```

i'm don't using brew

**Comment by janpio** [comment]
@GlebBelyaev We need the complete output you are getting. The relevant bits are missing in what you posted.

**Comment by PatrickDotStar** [comment]
Hello
I am experiencing the same issue. I haven't touched my project for a while but after updating to the latest fastlane my `gym` does not work anymore

Ruby version I use
`ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin18]`

<details>
<summary>Summary</summary>

```
[11:13:41]: Exit status: 126

+---------------+-------------------------+
|            Build environment            |
+---------------+-------------------------+
| xcode_path    | /Applications/Xcode.app |
| gym_version   | 2.116.0                 |
| export_method | app-store               |
| sdk           | iPhoneOS12.1.sdk        |
+---------------+-------------------------+

+---------------------+----------+
|          Lane Context          |
+---------------------+----------+
| DEFAULT_PLATFORM    | ios      |
| PLATFORM_NAME       | ios      |
| LANE_NAME           | ios beta |
| LATEST_BUILD_NUMBER | 56       |
| BUILD_NUMBER        | 57       |
+---------------------+----------+

+------+----------------------------+-------------+
|                fastlane summary                 |
+------+----------------------------+-------------+
| Step | Action                     | Time (in s) |
+------+----------------------------+-------------+
| 1    | Verifying fastlane version | 0           |
| 2    | default_platform           | 0           |
| 3    | app_store_build_number     | 10          |
| 4    | increment_build_number     | 2           |
| 💥   | gym                        | 63          |
+------+----------------------------+-------------+

[11:13:41]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

➡️   undefined method `each' for nil:NilClass
    https://github.com/fastlane/fastlane/issues/13959 [open] 9 💬
    a week ago

➡️  Rate limit has been exceeded for: testflight|default|APPLE_CONNECT
    https://github.com/fastlane/fastlane/issues/12845 [open] 56 💬
    a day ago

➡️  Gateway error when downloading reviews
    https://github.com/fastlane/fastlane/issues/14022 [open] 7 💬
    16 hours ago

and 148 more at: https://github.com/fastlane/fastlane/search?q=undefined%20method%20%60each%27%20for%20nil&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.
bundler: failed to load command: fastlane (/usr/local/bin/fastlane)
NoMethodError: [!] undefined method `each' for nil:NilClass
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/gym/lib/gym/error_handler.rb:147:in `print_full_log_path'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/gym/lib/gym/error_handler.rb:59:in `handle_build_error'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/gym/lib/gym/runner.rb:97:in `block in build_app'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane_core/lib/fastlane_core/command_executor.rb:89:in `execute'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/gym/lib/gym/runner.rb:93:in `build_app'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/gym/lib/gym/runner.rb:18:in `run'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/gym/lib/gym/manager.rb:19:in `work'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/actions/build_ios_app.rb:64:in `run'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/runner.rb:253:in `block (2 levels) in execute_action'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/runner.rb:232:in `block in execute_action'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/runner.rb:227:in `chdir'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/runner.rb:227:in `execute_action'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/fast_file.rb:154:in `method_missing'
  Fastfile:38:in `block (2 levels) in parsing_binding'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/lane.rb:33:in `call'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/lane_manager.rb:56:in `cruise_lane'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
  /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
  /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
  /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
  /Library/Ruby/Gems/2.3.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/commands_generator.rb:333:in `run'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/fastlane/lib/fastlane/cli_tools_distributor.rb:113:in `take_off'
  /Library/Ruby/Gems/2.3.0/gems/fastlane-2.116.0/bin/fastlane:23:in `<top (required)>'
  /usr/local/bin/fastlane:23:in `load'
  /usr/local/bin/fastlane:23:in `<top (required)>'
```

</details>

**Comment by SharudAgarwal** [comment]
I am getting this error with fastlane 2.116.1. I reverted back to fastlane 2.108.0 and there are no errors. Here is my entire gemfile diff:

```
remote: https://rubygems.org/
   specs:
     CFPropertyList (3.0.0)
-    addressable (2.6.0)
+    addressable (2.5.2)
       public_suffix (>= 2.0.2, < 4.0)
     atomos (0.1.3)
     babosa (1.0.2)
@@ -13,29 +13,28 @@ GEM
       highline (~> 1.7.2)
     declarative (0.0.10)
     declarative-option (0.1.0)
-    digest-crc (0.4.1)
     domain_name (0.5.20180417)
       unf (>= 0.0.5, < 1.0.0)
-    dotenv (2.7.1)
-    emoji_regex (1.0.1)
+    dotenv (2.5.0)
+    emoji_regex (0.1.1)
     excon (0.62.0)
-    faraday (0.15.4)
+    faraday (0.15.3)
       multipart-post (>= 1.2, < 3)
     faraday-cookie_jar (0.0.6)
       faraday (>= 0.7.4)
       http-cookie (~> 1.0.0)
-    faraday_middleware (0.13.1)
+    faraday_middleware (0.12.2)
       faraday (>= 0.7.4, < 1.0)
-    fastimage (2.1.5)
-    fastlane (2.116.1)
+    fastimage (2.1.4)
+    fastlane (2.108.0)
       CFPropertyList (>= 2.3, < 4.0.0)
       addressable (>= 2.3, < 3.0.0)
       babosa (>= 1.0.2, < 2.0.0)
-      bundler (>= 1.12.0, < 3.0.0)
+      bundler (>= 1.12.0, < 2.0.0)
       colored
       commander-fastlane (>= 4.4.6, < 5.0.0)
       dotenv (>= 2.1.1, < 3.0.0)
-      emoji_regex (>= 0.1, < 2.0)
+      emoji_regex (~> 0.1)
       excon (>= 0.45.0, < 1.0.0)
       faraday (~> 0.9)
       faraday-cookie_jar (~> 0.0.6)
@@ -43,7 +42,6 @@ GEM
       fastimage (>= 2.1.0, < 3.0.0)
       gh_inspector (>= 1.1.2, < 2.0.0)
       google-api-client (>= 0.21.2, < 0.24.0)
-      google-cloud-storage (>= 1.15.0, < 2.0.0)
       highline (>= 1.7.2, < 2.0.0)
       json (< 3.0.0)
       mini_magick (~> 4.5.1)
@@ -64,7 +62,7 @@ GEM
       xcodeproj (>= 1.6.0, < 2.0.0)
       xcpretty (~> 0.3.0)
       xcpretty-travis-formatter (>= 0.0.3)
-    fastlane-plugin-changelog (0.13.0)
+    fastlane-plugin-changelog (0.12.0)
     gh_inspector (1.1.3)
     google-api-client (0.23.9)
       addressable (~> 2.5, >= 2.5.1)
@@ -74,15 +72,6 @@ GEM
       representable (~> 3.0)
       retriable (>= 2.0, < 4.0)
       signet (~> 0.9)
-    google-cloud-core (1.3.0)
-      google-cloud-env (~> 1.0)
-    google-cloud-env (1.0.5)
-      faraday (~> 0.11)
-    google-cloud-storage (1.16.0)
-      digest-crc (~> 0.4)
-      google-api-client (~> 0.23)
-      google-cloud-core (~> 1.2)
-      googleauth (>= 0.6.2, < 0.10.0)
     googleauth (0.6.7)
       faraday (~> 0.12)
       jwt (>= 1.4, < 3.0)
@@ -94,7 +83,7 @@ GEM
     http-cookie (1.0.3)
       domain_name (~> 0.5)
     httpclient (2.8.3)
-    json (2.2.0)
+    json (2.1.0)
     jwt (2.1.0)
     memoist (0.16.0)
     mime-types (3.2.2)
@@ -107,7 +96,7 @@ GEM
     nanaimo (0.2.6)
     naturally (2.2.0)
     os (1.0.0)
-    plist (3.5.0)
+    plist (3.4.0)
     public_suffix (2.0.5)
     representable (3.0.4)
       declarative (< 0.1.0)
@@ -131,15 +120,15 @@ GEM
       unicode-display_width (~> 1.1, >= 1.1.1)
     tty-cursor (0.6.0)
     tty-screen (0.6.5)
-    tty-spinner (0.9.0)
-      tty-cursor (~> 0.6.0)
+    tty-spinner (0.8.0)
+      tty-cursor (>= 0.5.0)
     uber (0.1.0)
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.7.5)
-    unicode-display_width (1.4.1)
+    unicode-display_width (1.4.0)
     word_wrap (1.0.0)
-    xcodeproj (1.8.1)
+    xcodeproj (1.7.0)
       CFPropertyList (>= 2.3.3, < 4.0)
       atomos (~> 0.1.3)
       claide (>= 1.0.2, < 2.0)
```

**Comment by PatrickDotStar** [comment]
Ok it looks like I was able to fix it however I am not exactly sure how🙈 
I guess what it really fixed it is that I updated the RubyGems system software to 3.0.2. (I was using 2.5.x if I remember correctly)
You can check your RubyGems version with `gem -v`
If you want to update you just need to run `gem update --system`. You may need to run this as `sudo`
However I am not exactly sure if this really fixed it but I guess so because it also fixed [another issue](https://github.com/fastlane/fastlane/issues/12296) I had when using fastlane with bundler.
Now if I run `bundle exec fastlane gym` again it runs with no errors. (I tried it with the same fastlane version I had the error with so fastlane shouldn't be the issue)
Would be great if anyone could try this out and confirm that this solves the issue🙂

**Comment by joelgetaction** [comment]
I am also seeing this issue in `gym` after updating to fast lane 2.117.1.  Any ideas how to fix this?

**Comment by joelgetaction** [comment]
Updating `gem` to 3.0.3 did ***not*** help.  Still the same error ... Any ideas?

**Comment by joelgetaction** [comment]
OK, I see this in the `gym` output and it may be the root cause, but the `each` error on nil is obscuring this.  So it appears that there's a bug in my config (maybe since I updated Xcode?) and also a bug in gym or fast lane's error reporting:
```
xcodebuild: error: SDK "iOS 11.4" cannot be located.
```

**Comment by PatrickDotStar** [comment]
@joelgetaction this looks like your system is missing the iOS 11.4 SDK. You should check with Xcode if you have installed this SDK otherwise it won‘t build.

**Comment by joelgetaction** [comment]
Thanks @PatrickDotStar .  I hadn't built for a long time, especially since upgrading to Xcode 10.  So I updated the SDK and reverted to the legacy build system since I'm still on RN 0.57.1:
```ruby
sdk = "iphoneos12.1"

...

gym(
  ...
  xcargs: "-UseModernBuildSystem=NO",
)
```

Now it's working!

And by the way, I was able to get the build working on my main admin account with 2FA already enabled by just creating an app-specific password with Apple for that account.  So that seems to work nicely!
